engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/task/PotentialJobTaskProcessor.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.engine.processing.bpmn.task;
+
+import io.camunda.zeebe.engine.processing.bpmn.BpmnElementContext;
+import io.camunda.zeebe.engine.processing.bpmn.BpmnElementProcessor;
+import io.camunda.zeebe.engine.processing.bpmn.behavior.BpmnBehaviors;
+import io.camunda.zeebe.engine.processing.bpmn.behavior.BpmnStateTransitionBehavior;
+import io.camunda.zeebe.engine.processing.deployment.model.element.ExecutableJobWorkerTask;
+
+public abstract class PotentialJobTaskProcessor<T extends ExecutableJobWorkerTask>
```

Comment:
:wrench: How about we name this class so that it's clear that the processors implementing it support Job Worker implementations?

Maybe:
* `JobWorkerTaskSupportingProcessor`
* `JobWorkerTaskSupportedProcessor`
* `JobWorkerTaskEnabledProcessor`

It also fits with the naming convention for processors we discussed [here](https://camunda.slack.com/archives/C037W9NMATG/p1699439287876019).

Reply:
Thanks, @koevskinikola, I like the suggestions 👍 
I'll go with option 1, `JobWorkerTaskSupportingProcessor`. Sounds closest to what the processor does/is to me.